qiita_ware/ebi.py

```diff
@@ -729,20 +710,20 @@ def generate_curl_command(
           be generated before executing this function
         """
         # make sure that the XML files have been generated
-        url = '?auth=ENA%20{0}%20{1}'.format(ebi_seq_xfer_user,
-                                             ebi_access_key)
-        curl_command = (
-            'curl {0}-F "SUBMISSION=@{1}" -F "STUDY=@{2}" -F "SAMPLE=@{3}" '
-            '-F "RUN=@{4}" -F "EXPERIMENT=@{5}" "{6}"'
-        ).format(
-            '-k ' if ebi_skip_curl_cert else '',
-            self.submission_xml_fp,
-            self.study_xml_fp,
-            self.sample_xml_fp,
-            self.run_xml_fp,
-            self.experiment_xml_fp,
-            join(ebi_dropbox_url, url)
-        )
+        url = '?auth=ENA%20{0}%20{1}'.format(quote(ebi_seq_xfer_user),
+                                             quote(ebi_seq_xfer_pass))
+        curl_command = 'curl -sS -k'
```

Comment:
Usually is recommended to create a list of strings and use appends and a `''.join()` at the end. IIRC, string concatenation is more expensive than list append.